File: client-service/backend/client-service/src/main/java/com/cinema/clientservice/web/requests/ReservationDetails.java
package com.cinema.clientservice.web.requests;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.time.LocalTime;
import java.util.List;

public record ReservationDetails(
        String movieTitle,
        String languageVersionName,
        LocalDate date,
        LocalDateTime time,
        List<TicketReservationDetailsDto> tickets,
        double total
) {
}

File: client-service/frontend/client-service/src/app/models/reservationDetails.ts
import {TicketReservationDetailsDto} from "./ticketReservationDetailsDto";

export interface ReservationDetails {
  movieTitle: string,
  languageVersionName: string,
  date: Date,
  time: Date,
  tickets: TicketReservationDetailsDto[];
  total: number;
}
